### DISCUSSION TITLE
Add the Ricoh/JVC controller to the Epson Equity LT laptop

### DISCUSSION THREAD

**Discussion by Fenix770** [86Box/86Box#4041] [Ideas]
**Is your feature request related to a problem? Please describe.**
No. I think the emulation of this machine is not complete. The HDD included in this machine was the same that Victor V86P and Toshiba T1200, this three machines have an 8086 CPU and this particular HDD controller was used on the XT laptop machines.
But the issue is the HDD is recognized by Epson DOs 3.20, but throws a "Disk boot failure" at startup after the installation of 
MS-DOS, like the Toshiba T1200 before the implementation of this interphace as internal.

This site show a list of laptop XTs with the Ricoh/JVC HDD interphase. 
https://knm.org.uk/blog/2017/04/the-jvc-26-pin-hard-disk-interface-part-1/

On the other hand this machine seems to use the blue backlight screen very similar to the used on the T1200 machine, and also
seems to have screen overscan (the borders of the screen have no signal)
Here is a Youtube video.
https://www.youtube.com/watch?v=YTW3d0qTA0U

**Comment by OBattler** [comment]
Converted it to a discussion because this is a feature request that requires an undumped ROM (the hard disk controller ROM) to be dumped and it is therefore unfeasible until that is done.

**Comment by flama12333** [comment]
We have the hdd rom dumped by someone at vcfed.

**Comment by flama12333** [comment]
here is the screenshot
<img width="640" height="400" alt="Monitor_1_20260223-193325-581" src="https://github.com/user-attachments/assets/f7aff5b7-d83e-417f-87a1-484503077e84" />
<img width="640" height="400" alt="Monitor_1_20260223-193333-648" src="https://github.com/user-attachments/assets/f5229ef7-b070-42b1-b05f-b65046517669" />
